code/modules/vending/clothingbooth.dm

```diff
@@ -65,6 +62,9 @@ var/list/clothingbooth_items = list()
 		if (user.stat != 0 || user.getStatusDuration("stunned"))
 			return
 		src.set_open(1)
+		sleep(2 SECONDS)
```

Comment:
I'm not sure if calling `sleep` in relaymove is a good idea.

Reply:
ill look into it :heart: